Cargo.toml

```diff
@@ -28,10 +29,15 @@ secp256k1 = "0.19.0"
 bitcoinconsensus = { version = "0.19.0-1", optional = true }
 serde = { version = "1", optional = true }
 
+[dependencies.schemars]
+version = "0.8.0"
+optional = true
+
 [dev-dependencies]
-serde_derive = "<1.0.99"
-serde_json = "<1.0.45"
+serde_derive = "<1.0.99, >= 1.0.0"
```

Comment:
For this "~1.0.0" exists

Reply:
Not sure I understand what type of problem you are trying to prevent by limiting patch version number from going above 100. Rather, "<1.0.99, >= 1.0.0" introduce a problem, such that serde crate having patches >100 already will stop updating.

Reply:
I didn't introduce the <1.0.99 constraint so don't ask me; I'm merely fixing the lack of lower bound issue! I'm stupid and I don't understand the balls in the air and am introducing a fix. Please see https://github.com/rust-bitcoin/rust-bitcoin/pull/495 for further discussion as this is unrelated to this patch in particular.

https://jubianchi.github.io/semver-check/#/%3C1.0.99/0.1.2

Reply:
got you, sorry missed that it was not introduced by you and is unrelated